python/src/trezorlib/_internal/firmware_headers.py

```diff
@@ -53,12 +53,19 @@ def all_zero(data: bytes) -> bool:
     return all(b == 0 for b in data)
 
 
-def _check_signature_any(fw: "SignableImageProto", is_devel: bool) -> Status:
+def _check_signature_any(fw: "SignableImageProto", is_devel: bool = False) -> Status:
+    print("checking signature of:", fw)
```

Comment:
This prints the whole image, including the (binary) code, please limit the scope of printing or remove entirely

Reply:
yeah this is a debug statement that shouldn't be there :( e9365f